config.yaml

```diff
@@ -54,3 +54,150 @@ options:
     description: |
       Public url of Jenkins frontend, including the hostname and prefix. This
       is used by Jenkins whenever generating full links.
+  jenkins-io-apt-key:
```

Comment:
Instead of making this a new config key, what do you think to keep it hard-wired in the code for now? (in the same way as the APT deb line is, in the APT_SOURCE variable of packages.py).

The reason is that I'd like to see either:

1.  Two generic "source" and "key" config options to specify the non-standard APT source to fetch the jenkins package from. Those options would default to the official APT repository, but the config keys themselves would not be named after it (e.g. "key" vs "jenkins-io-apt-key").

2. Keep the code as it is now, where you cannot really specify an alternative APT source (you can only point to an URL), and everything is hard-coded.

Clearly, 1 is more desirable, but till we get there I'd stick with 2, otherwise we end up with a mixed situation like the one in this branch.